src/docs/sdk/event-payloads/release.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Detect Release Automatically
+---
+
+<Alert title="Note on --- " level="info">
+  Should we call special attention to something to encourage best practices and prevent and bad DX?
+</Alert>
+
+
+## Release
+
+
+```json
+{
+  "release": "my-project-name@1.0.0"
+}
+```
+
+```json
+{
+  "release": "721e41770371db95eee98ca2707686226b993eda"
+}
+```
+
+## SDK Examples
+
+| Source        | Description                                                                                                                     | <div style="width:220px">Examples</div>               |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| Python SDK    | -SENTRY_RELEASE env variable <br /> - Output of git rev-parse HEAD                                                                                                                                |                                                       |
+| Rust SDK      |                                                                                                                                 |                                                       |
+| Browser SDK   |                                                                                                                                 |                                                       |
+| Node SDK      |                                                                                                                                 |                                                       |
+| Dart/Flutter  |                                                                                                                                 |                                                       |
+| Java/Android  |                                                                                                                                 |                                                       |
```

Comment:
Java will be different than Android

Reply:
https://www.notion.so/sentry/PR-Dev-Auto-release-setting-techniques-a0da5ba23058406ca56be5fe3524d4c4

yep you are right they are separated in the second column here, should just give them both a row, and make android (Java) or something